sale_order_line_clone/models/sale_order.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    can_clone_sale_line = fields.Boolean(
+        string="Can Clone Sale Order Line",
+        default=True,
+        compute="_compute_product_sale_line",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("product_id", "product_id.can_clone_sale_line")
+    def _compute_product_sale_line(self):
+        for record in self:
+            if record.product_id:
+                prod_can_clone_sale_line = record.product_id.can_clone_sale_line
+                record.write({"can_clone_sale_line": prod_can_clone_sale_line})
```

Comment:
Same as above, why do you use the write?

Reply:
Changed